telebot/types.py

```diff
@@ -1208,6 +1214,13 @@ def de_json(cls, json_string):
         if 'story' in obj:
             opts['story'] = Story.de_json(obj['story'])
             content_type = 'story'
+        if 'external_reply' in obj:
+            opts['external_reply'] = ExternalReplyInfo.de_json(obj['external_reply'])
+            content_type = 'text' # @Badiboy not sure about content_types in here, please check
```

Comment:
@Badiboy check this line and the lines below

Reply:
You can check it in previous commit

Reply:
As I get the meaning of the field "external_reply" - it contains ADDITIONAL INFORMATION to the main content. So it should not process content_type at all: content_type will be set by the main data fields like "text"/"audio" etc.

The same for "quote".

I'll removed the content_type assignment in commit.